src/client/components/style/labels.js

```diff
@@ -55,12 +55,9 @@ export function stylePropsToLabelPos(h, v) {
 
 
 export function PositionButton({ value, selected, onClick }) {
-  const style = { cursor:'pointer', display:'flex', alignItems:'center', justifyContent:'center' };
```

Comment:
So as a general guideline, we should avoid using inline style objects and prefer using css classes?

Reply:
Yeah what is the best practice here? I find inline styles to be much more convenient because I don't have to go to another file and use the IDEs find function to find the style class. But inline styles can get unweildy when they get big.

Reply:
The general best practice for 20+ years has been to use CSS classes as a rule.  The main exception for when you'd use `style` is where a value is clearly dependent on the model -- the colour swatch `background-color` is a good example of that.

It's trendy with some React people to use `style` attributes for everything.  You could, but is it good to use that for everything by default?  I certainly couldn't have done what you see in the video without using CSS proper.

We don't *have* to use CSS proper, but I'd strongly suggest it.  Generally, the way to create content for apps like this is to create the HTML content templates, mapped from the model, and then to create all the CSS afterwards.  Then, you only need one file open at a time.  Alternatively, you could use two panels in your editor.  VSC supports that.  A big benefit of using CSS proper is that you get live, instant previews of your CSS changes.  That's really easy, fast, and productive.

What do you think?

Reply:
Makes sense to me.  I think it is important to stick to the suggested conventions to make things more consistent in the codebase.